app/code/Magento/InventoryInStorePickup/Model/ResourceModel/OrderPickupPoint/GetPickupPointByOrderId.php

```diff
@@ -0,0 +1,60 @@
+<?php
+/**
+ * Copyright © Magento, Inc. All rights reserved.
+ * See COPYING.txt for license details.
+ */
+declare(strict_types=1);
+
+namespace Magento\InventoryInStorePickup\Model\ResourceModel\OrderPickupPoint;
+
+use Magento\Framework\App\ResourceConnection;
+
+/**
+ * Class GetPickupPointByOrderId.
```

Comment:
Repeating class name in the comment is redundant. Clear responsibility description is sufficient.

Reply:
Done.